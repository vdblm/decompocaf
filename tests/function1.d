void f(int fe) {
  Print(fe);
}

void ff() {
 f(34);
}

void g(int a, string b, bool c) {
  Print(c, b, a);

}

int main() {
  f(12);
  ff();
  g(2, " and ",true);
  f(10 + 3 % 2 + 5);
}