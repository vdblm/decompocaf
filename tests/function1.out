12
34
true and 2
16
